.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-ci
+repo_token: sJbZM38EkByCbBH5oHH19QiX4PckDx0Mu
```

Comment:
Was this part of this PR

Reply:
No, it was not, I rebased with develop when I got merge conflicts after you had merged the two pending PR's to develop and so I got the .coveralls.yml file.